cypress/integration/Taxonomy/subject_spec.js

```diff
@@ -6,61 +6,64 @@
+      `${taxonomyApi}/subjects/${selectSubject}/topics?recursive=true&language=nb`,
       'allSubjectTopics',
     );
-    cy.apiroute('GET', '/taxonomy2/v1/subjects/urn:subject:12/filters', 'allSubjectFilters');
+    cy.apiroute('GET', `${taxonomyApi}/subjects/${selectSubject}/filters`, 'allSubjectFilters');
 
-    cy.visit('/structure/urn:subject:12');
-    cy.wait(['@allSubjects', '@allSubjectTopics', '@allSubjectFilters']);
+    cy.visit(`/structure/${selectSubject}`);
+    cy.apiwait(['@allSubjects', '@allSubjectTopics', '@allSubjectFilters']);
   });
 
   beforeEach(() => {
     setToken();
   });
 
   it('should add a new subject', () => {
-    cy.intercept('POST', '/taxonomy2/v1/subjects', []).as('addSubject');
+    cy.intercept('POST', `${taxonomyApi}/subjects`, []).as('addSubject');
 
     cy.get('[data-testid=AddSubjectButton]').click();
     cy.get('[data-testid=addSubjectInputField]').type('Cypress test subject{enter}');
-    cy.wait('@addSubject');
+    cy.apiwait('@addSubject');
   });
 
   it('should have a settings menu where everything works', () => {
-    cy.intercept('PUT', `/taxonomy2/v1/subjects/urn:subject:12`, []).as('newSubjectName');
-    cy.intercept('POST', '/taxonomy2/v1/topics', []).as('addNewTopic');
-    cy.intercept('POST', '/taxonomy2/v1/filters', []).as('addFilter');
-    cy.intercept('PUT', '**/taxonomy2/v1/filters/*', []).as('editFilter');
-    cy.intercept('DELETE', '**/taxonomy2/v1/filters/*', []).as('deleteFilter');
-    cy.apiroute('GET', '/taxonomy2/v1/topics?language=nb', 'allTopics');
-    cy.intercept('POST', '/taxonomy2/v1/topic-filters',[]);
-    cy.intercept('POST', '/taxonomy2/v1/subject-topics', []).as('addNewSubjectTopic');
+    cy.intercept('PUT', `${taxonomyApi}/subjects/${selectSubject}`, []).as('newSubjectName');
+    cy.intercept('POST', `${taxonomyApi}/topics`, []).as('addNewTopic');
+    cy.intercept('POST', `${taxonomyApi}/filters`, []).as('addFilter');
+    cy.intercept('PUT', `${taxonomyApi}/filters/*`, []).as('editFilter');
+    cy.intercept('DELETE', `${taxonomyApi}/filters/*`, []).as('deleteFilter');
+    cy.apiroute('GET', `${taxonomyApi}/topics?language=nb`, 'allTopics');
+    cy.intercept('POST', `${taxonomyApi}/topic-filters`,[]);
+    cy.intercept('POST', `${taxonomyApi}/subject-topics`, []).as('addNewSubjectTopic');
 
     cy.get('[data-cy=settings-button-subject]')
       .first()
       .click();
-    cy.wait('@allTopics')
+    cy.apiwait('@allTopics')
     cy.get('[data-testid=changeSubjectNameButton]').click();
     cy.get('[data-testid=inlineEditInput]').type('TEST{enter}');
-    cy.wait('@newSubjectName');
+    cy.apiwait('@newSubjectName');
 
     cy.get('[data-cy=settings-button-subject]')
       .first()
       .click();
     cy.get('[data-testid=editSubjectFiltersButton]').click();
     cy.get('[data-testid=addFilterButton]').click();
     cy.get('[data-testid=addFilterInput]').type('cypress-test-filter{enter}');
-    cy.wait('@addFilter');
```

Comment:
Jeg ser at du setter `addFilter`, `allSubjectFilters`, `addSubject`-fixtures til `[]`, og hvis du vil at den alltid skal være `[]` så burde du la `cy.wait` stå. Bytter du om til `cy.apiwait` til vi få ekte data alle de stedene neste gang noen kjører `cy-record`

Reply:
Det er sant. Eg berre såg at alle disse kalla gikk til faktisk backend, så bytte til apiwait gjorde at fixtures blei brukt. Kan teste å endre tilbake.

Reply:
Det er ingen praktisk betydning mellom `cy.wait` og `cy.apiwait` (dersom man ikke kjører `cy-record`) for `cy.apiwait` har en sjekk om man kjører `cy-record` og hvis ikke så kjører den `cy.wait`. Tror det er lurt å bruke `cy.wait` på de dataene vi faktisk vil holde constant, så ingen innfører noe uforventet feil neste gang man kjører `cy-record`

Reply:
Oppdatert pr til å mocke alle kall til tax som tidligere gikk direkte. Pluss en ekstra test.